packages/neon_framework/lib/src/blocs/weather_status.dart

```diff
@@ -50,6 +50,12 @@ class _WeatherStatusBloc extends InteractiveBloc implements WeatherStatusBloc {
     });
 
     timer = TimerBloc().registerTimer(const Duration(minutes: 5), refresh);
+
+    location.listen((locationResult) async {
+      if (!locationResult.isLoading && !locationResult.hasError) {
```

Comment:
```suggestion
      if (!locationResult.isLoading && locationResult.hasData) {
```
The error might be because the user is offline, but cached data might be available.